docker/release/nanoserver/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # escape=`
-FROM microsoft/nanoserver
+FROM microsoft/nanoserver:latest
 MAINTAINER brycem@microsoft.com
```

Comment:
Does that go anywhere?  All the rest have actual email addresses on them.  If we want to update them, that should probably be a separate item.  The MAINTAINER tag is deprecated anyway.  I filed #3465 for this.  Can we continue this discussion there?